### DISCUSSION TITLE
Is it possible to open a session in the background?

### DISCUSSION THREAD

**Discussion by y-mx-b** [zellij-org/zellij#2040] [Q&A]
I wrote a simple shell script to open a bunch of tmux sessions for various projects that I'm working on. In switching over to zellij, I was attempting to replicate this functionality, but I'm at a loss as for how to open a session in the background. Is there a way to do this? I didn't see anything in the documentation or in the help message either.

**Comment by imsnif** [comment]
I don't think there's a way to do this, but I'd be open to merging this functionality if you or someone else would like to work on this.

**Reply by y-mx-b** [reply]
Opened #2049. It's slightly bugged at the moment, but I'm busy and it's late, so I'll have to take another look at some other point.

**Reply by imsnif** [reply]
Cool - unless there are specific questions, I usually prefer to take a look at a PR when it's flagged as complete. So let me know when you think it's ready.

**Reply by y-mx-b** [reply]
> Cool - unless there are specific questions, I usually prefer to take a look at a PR when it's flagged as complete. So let me know when you think it's ready.

I've fixed the bug I've mentioned. `cargo x test` shows all tests passing. I believe it's ready now.
